packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_databases_service_integration_add_confirm": "Confirmer",
+  "pci_databases_service_integration_add_cancel": "Annuler",
+  "pci_databases_service_integration_add": "Ajouter un Kafka",
```

Comment:
```suggestion
  "pci_databases_service_integration_add": "Ajouter un service Kafka",
```

Reply:
I cannot change the text as it has been already validated by content team.